spec/game_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+RSpec.describe Game do
+  let(:game) { described_class.new }
+  #let will create a local variable named game that is initialized with a new instance of the Game class. 
+  #(:game) is the name of the variable that will be created.
+  #{described_class.new} is the value that the variable will be initialized with.
+
+  describe '#initialize' do
+    it 'initializes with game_over set to false' do #tests that the Game class initializes with game_over set to false
+      expect(game.instance_variable_get(:@game_over)).to eq(false)
+    end
+  end
+
+  describe '#display_main_menu' do
+    before do
+      allow(game).to receive(:gets).and_return('q')
+    end
+
+    it 'displays the main menu' do #tests that the display_main_menu method displays the main menu
+      expect { game.display_main_menu }.to output("Welcome to BATTLESHIP\nEnter 'p' to play. Enter 'q' to quit.\n").to_stdout
+    end
+  end
+
+  describe '#handle_main_menu_input' do #tests that the handle_main_menu_input method correctly handles input
```

Comment:
There are a few reasons we don't need these tests here; first, they deal with user input so we can do all of the testing ourselves. Second, these methods are private so they are only called within the class itself. As long as we test all of the public methods completely, that will necessarily also cover these private methods